indexer/feature_impl.cpp

```diff
@@ -65,12 +65,12 @@ bool IsHouseNumberDeepCheck(strings::UniString const & s)
 
 bool IsHouseNumber(string const & s)
 {
-  return (!s.empty() && IsDigit(s[0]));
+  return (!s.empty() && (IsDigit(s.front()) || IsDigit(s.back())));
 }
 
 bool IsHouseNumber(strings::UniString const & s)
 {
-  return (!s.empty() && IsDigit(s[0]));
+  return (!s.empty() && (IsDigit(s.front()) || IsDigit(s.back())));
```

Comment:
Строение 22
elb.23

> On Apr 8, 2016, at 12:51, Viktor Govako <notifications@github.com> wrote:
> 
> In indexer/feature_impl.cpp <https://github.com/mapsme/omim/pull/2842#discussion_r59003152>:
> 
> >  }
> >  
> >  bool IsHouseNumber(strings::UniString const & s)
> >  {
> > -  return (!s.empty() && IsDigit(s[0]));
> > +  return (!s.empty() && (IsDigit(s.front()) || IsDigit(s.back())));
> Саша, а вот тут причина правки IsDigit(s.back()) непонятна.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2842/files/adf7724a884ebdd02d93d6b2a7083debc55f28c9#r59003152>